boards/arm/nrf5340pdk_nrf5340/nrf5340pdk_nrf5340_cpuapp.dts

```diff
@@ -14,5 +14,6 @@
 		zephyr,flash = &flash0;
 		zephyr,code-partition = &slot0_partition;
 		zephyr,sram-secure-partition = &sram0_s;
+		zephyr,sram-non-secure-partition = &sram0_ns;
```

Comment:
There's some (inconsistent) precedent for documenting standard `chosen` properties in `doc/guides/dts/legacy-macros.rst`.  The surrounded text suggests that it should be complete, so these should be added to it, replacing or augmenting the existing `zephyr,sram` entry.

@mbolivar-nordic if this section of that module remains relevant after we remove support for legacy macros perhaps it should move elsewhere before release?

Reply:
> @mbolivar-nordic if this section of that module remains relevant after we remove support for legacy macros perhaps it should move elsewhere before release?

That's a good idea. I have added the following to my TODO list for v2.3 DT docs:

- centralize documentation of chosen zephyr nodes to a non-legacy file, provide a reference to them from the intro

If you're OK with letting me take care of that after freeze, I will do so.

Reply:
@pabigot ^^
(I am ok, let me know if there's anything in need to do here)

Reply:
@pabigot ^^

Reply:
I'm fine with @mbolivar-nordic doing this later.

Reply:
thanks @pabigot